README.md

```diff
@@ -70,6 +70,8 @@ please read the file [build_system.md](documentation/build_system.md).
 | [HiKey Board (HiSilicon Kirin 620)](https://www.96boards.org/products/hikey/)|`PLATFORM=hikey`|
 | MediaTek MT8173 EVB Board|`PLATFORM=mediatek-mt8173`|
 | Texas Instruments DRA7xx|`PLATFORM=ti-dra7xx`|
+| FSL ls1021a|`PLATFORM=ls-ls1021atwr`|
```

Comment:
Even though the board is easily googled, it would be helpful with a link describing the board here.